web/packages/teleport/src/Discover/Database/DeployService/AutoDeploy/SelectSubnetIds.tsx

```diff
@@ -174,11 +174,9 @@ export function SelectSubnetIds({
                 <Icons.Refresh size="medium" mr={2} /> Refresh
               </ButtonSecondary>
             </HoverTooltip>
-            {selectedSubnetIds.length > 0 && (
-              <P3>
-                <b>Num Selected Subnets</b>: {selectedSubnetIds.length}
-              </P3>
-            )}
+            <P3>
+              {`${selectedSubnetIds.length} ${pluralize(selectedSubnetIds.length, 'subnet')} selected`}
```

Comment:
FYI, I just submitted this small PR which fixes my mistake in `pluralize`. ;) https://github.com/gravitational/teleport/pull/44949

Since the count is no longer hidden behind a zero check, AFAIK it should say "0 subnets selected" instead of "0 subnet selected".